webapp/src/components/editableCell.tsx

```diff
@@ -32,47 +29,43 @@ export const EditableCell = ({
     }
   };
 
+  // Respond to any external changes in the input field
+  // (I don't know if we actually need this)
+  useEffect(() => {
+    setValue(cell.value);
+  }, [cell.value]);
+
   // Wait to actually update the table state when we unfocus the input field
   const onBlur = () => {
+    console.log(row.index, column.Header, value);
     updateData(row.index, column.Header, value);
-
-    // This is the callback that will do in-line validation
-    const errors = preValidate(value);
-
-    if (errors.length > 0) {
-      setAppliedStyle(errorStyle);
-      // Show errors in any other way cuz of pre-validation
-    }
   };
 
-  function applyPostValidationErrors() {
+  function applyValidationErrorStyles(): Object {
     console.log("handlepostvalidate");
-    setAppliedStyle(notAnErrorStyle); // FIXME: This erases regex errors. Maybe we need just one source of errors
 
-    postValidationErrors.forEach((e) => {
-      if (e.index === row.index && e.column === column.Header?.toString()) {
-        console.log("post validation error hit");
-        setAppliedStyle(errorStyle);
-      }
-    });
-  }
+    // FIXME: force unwrap
```

Comment:
huh?

Reply:
Please add a more descriptive comment if you can. If there's pending work you can't finish until after the hack, add it to the project page.

Reply:
Comment revised